src/quo2/components/list_items/account/style.cljs

```diff
@@ -0,0 +1,112 @@
+(ns quo2.components.list-items.account.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn- background-color
+  [{:keys [state blur? customization-color]}]
+  (cond (and (or (= state :pressed) (= state :selected)) (not blur?))
+        (colors/custom-color customization-color 50 5)
+        (and (or (= state :pressed) (= state :selected)) blur?)
+        colors/white-opa-5
+        (and (= state :active) (not blur?))
+        (colors/custom-color customization-color 50 10)
+        (and (= state :active) blur?)
+        colors/white-opa-10
+        (and (= state :pressed) blur?) colors/white-opa-10
+        :else :transparent))
```

Comment:
Nitpick, but the first two conditions can be simplified so there's less repetition and it's easier to read
```clj
(or (= state :pressed) (= state :selected))
    (if blur? colors/white-opa-5 (colors/custom-color customization-color 50 5))
```

Same with the next ones
```clj
 (= state :active)
    (if blur? colors/white-opa-10 (colors/custom-color customization-color 50 10))
```

Reply:
Sure thing, done!